dapps/digital-twin-detail/src/components/container/Entry.ts

```diff
@@ -32,22 +32,20 @@ export default class ContainerEntryComponent extends mixins(EvanComponent) {
    */
   entrySchema: ObjectSchema;
 
-  /**
-   * Current value of the entry
-   */
-  value: any;
-
   created(): void {
-    const { dispatcherData, plugin, entries } = this.$store.state.container;
+    const { plugin } = this.$store.state.container;
     this.entrySchema = plugin.template.properties[this.name].dataSchema;
-    this.value = dispatcherData[this.name] || entries[this.name];
+  }
+
+  getValue(): any {
```

Comment:
use getter here? So the vue template doesn't need to be changed. `get value() { ... }`